packages/surimi/src/types/config.types.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Configuration options for Surimi.
+ */
+export interface SurimiConfig {
+  /**
+   * Enable strict CSS property type checking.
+   * When enabled, only valid CSS values are accepted (no arbitrary strings).
+   * @default false
+   */
+  strictProperties?: boolean;
+
+  /**
+   * Allow custom CSS properties (--*).
+   * @default true
+   */
+  allowCustomProperties?: boolean;
+
+  /**
+   * Allow vendor-prefixed CSS properties (-webkit-, -moz-, etc.).
+   * @default true
+   */
+  allowVendorPrefixes?: boolean;
```

Comment:
Also not checked, and we cannot type-check a runtime config setting. So this would lead to runtime errors, which I would not love